pages/18f/projects-partners/microrequests.md

```diff
@@ -4,6 +4,7 @@ questions:
   - microrequests
 redirect_from:
   - /microrequests/
+cSpell: ignore helpwanted
```

Comment:
since this is a channel name, should we backticks it as `#helpwanted` and remove the ignore here? (ditto for `writerlab.md`)

Reply:
✅ Done